macvmio/curie
# Split `ls` command into `images` and `ps`

Changed json formatting so the columns always have associated keys.

Test Plan:
- Ensure all CI checks pass
- Verify `curie images` lists all images
- Verify `curie ps` lists all containers
